content/04.use-cases-b.md

```diff
@@ -37,36 +37,34 @@ A CC BY 4.0 license specifies that any code (or other creative products) must be
+Periodic code, data and text review is useful for identifying errors early in the research process [@doi:10.1145/3341525.3387370], and informing further training and mentorship to fill gaps in skills.
+This is facilitated by a group of core features of Git and GitHub that allow contributors to discuss and simultaneously work on a project.
+For instance, users can clone and fork repositories (see [Box 1](#definitions)), allowing for modifications to be done on a linked copy of a repository, which can then later be merged into the main project through pull requests.
+Collaborators can comment on specific lines of code, text being changed, suggest changes, which can then be implemented with the click of a button, greatly facilitating peer review.
+Explicit project organization and increased communication within pull requests, in GitHub Issues, or in GitHub Discussions can help with project development and with potential merge conflicts due to users simultaneously working on the same sections.
 Moreover, version control [@https://sortee-github-hackathon.github.io/manuscript/v/latest/index.html#storing-and-archiving-version-controlled-data] allows researchers to make changes in code or documents without worrying about irreparably modifying someone else's work.
-By enabling more comprehensive remote collaboration, GitHub encourages the exchange of ideas among researchers at different institutions and in different countries, which can serve to improve the quality of the research itself by providing open access to data and code.
 
+By enabling more comprehensive remote collaboration, GitHub encourages the exchange of ideas among researchers at different institutions and in different countries, which can serve to improve the quality of the research itself by providing open access to data and code.
 
-### Writing a manuscript
+### Writing a manuscript{#writing-manuscript}
 
-<!--*contributors to this section:* Emma J. Hudgins, Katherine Hébert-->
+<!--*contributors to this section:* Emma J. Hudgins, Katherine Hébert, PHPB, Eric-->
 
 Beyond supporting collaborative code development, GitHub can be used for writing manuscripts.
 Writing a manuscript and storing its associated data and code in GitHub increases scientific reproducibility because text, code, and data can be found in one place.
-Writing a manuscript on GitHub may take more time when compared to using conventional text processors [@doi:10.1186/1751-0473-8-7].
-Nevertheless, GitHub has many features that can allow for a powerful collaborative workflow when writing manuscripts.
+Although more time-consuming when compared to using conventional text processors, GitHub has many features that can allow for a powerful collaborative workflow when writing manuscripts [@doi:10.1186/1751-0473-8-7].
```

Comment:
I'm not even sure that it's more time consuming once you factor in 5 collaborators all emailing you revisions on 3 different version of a Word doc.  It might be better to say something like "Using GitHub for collaborative writing may involve more initial setup and involves sacrificing some features of word processors...".  I'm ok with it as is though.

Reply:
This is a good point - focusing on the initial time invested is key here I think, because once the repo is set up and the co-authors know how to contribute and review the writing, it is hard to make a case that the process becomes longer. This little nuance would avoid giving the false impression that the whole writing process becomes heavier via GitHub, which might be a little discouraging.

Maybe a bridge between these two ideas, and one that surfaces in the conclusion of the cited paper, is: 

"Although it may involve more initial time investment, GitHub has many features that support a powerful collaborative workflow when writing manuscripts [@doi:10.1186/1751-0473-8-7]"

Reply:
Thank you both! I have added the suggestion to the file!